### PR TITLE
Add support for bzlmod

### PR DESCRIPTION
**Pull request by Bencodes**
Bringing in the initial support for Bzlmod. Another PR will follow integrating this into the Bazel central registry.

https://github.com/robolectric/robolectric-bazel/issues/50

### CODE REVIEW

File: examples/simple/WORKSPACE [hunk 1]
```diff
@@ -22,13 +22,13 @@ load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 bazel_skylib_workspace()
 
 http_archive(
-    name = "build_bazel_rules_android",
+    name = "rules_android",
```
Comment (Bencodes) [inline examples/simple/WORKSPACE:25]: This is named `rules_android` in the central registry.

**Review by utzcoz — APPROVED** [review]

**Comment by utzcoz** [comment]
@Bencodes Thanks. LGTM.

**Comment by utzcoz** [comment]
@Bencodes I have another question: is is possible to integrate robolectric-bazel to the Bazel central registry without releasing a new version of robolectric-bazel? I remember Bazel supports to use commit to integrate other libraries.

**Comment by utzcoz** [comment]
@Bencodes Oh, I think there will be a 4.10 relese recent days, what about integrating it to the central registry after 4.10 release? We can use the official release. cc @hoisie.

**Pull request merged**